README.rst

```diff
@@ -57,13 +57,31 @@ Django’s settings.py require the following format to connect to an Informix da
         'PASSWORD': 'passw0rd',
         'OPTIONS': {
             'DRIVER': '/path/to/iclit09b.so'. # Or iclit09b.dylib on macOS
+            'CPTIMEOUT': 120,
+            'ISOLATION_LEVEL': 'READ_UNCOMMITTED',
+            'LOCK_MODE_WAIT': 0,
         },
 	'TEST': {
 	    'NAME': 'myproject',
 	    'CREATE_DB': False
 	}
     }
 
+ISOLATION_LEVEL
+    This will set database isolation level at connection level
+    Possible values:
+        READ_COMMITED
+        READ_UNCOMMITTED
+        SERIALIZABLE
+
+LOCK_MODE_WAIT
+    This will set database LOCK MODE WAIT at connection level
+    Application can use this property to override the default server process for accessing a locked row or table.
```

Comment:
This line is >80 chars, which is longer than most terminals, which will make it harder to read. Please reformat.